core/arxiv/submission/process/process_source.py

```diff
@@ -58,6 +60,10 @@
 IN_PROGRESS: Status = 'in_progress'
 NOT_STARTED: Status = 'not_started'
 
+# Limits for reading stream of content
+SIZE_LIMIT = 15000000
```

Comment:
```suggestion
SIZE_LIMIT = 15_000_000
```
(just for readability per PEP515)